From da76ed3349fbd98cab515f83d5fd4fa7c5687010 Mon Sep 17 00:00:00 2001
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Mon, 5 Aug 2013 10:44:50 +0300
Subject: Create intel_sec aid for accessing Intel security devices

Issue: APDEV-504

Category: device enablement
Domain: AOSP.Security
Origin: internal
Upstream-Candidate: no, Intel specific group

Change-Id: I7995a9dc27ba2fe5ff6d1cd118e09f447b8282f9
---
 include/private/android_filesystem_config.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/private/android_filesystem_config.h b/include/private/android_filesystem_config.h
index 0ed0d78..f5a494d 100644
--- a/include/private/android_filesystem_config.h
+++ b/include/private/android_filesystem_config.h
@@ -91,6 +91,7 @@
 #define AID_NET_BW_STATS  3006  /* read bandwidth statistics */
 #define AID_NET_BW_ACCT   3007  /* change bandwidth statistics accounting */
 #define AID_NET_BT_STACK  3008  /* bluetooth: access config files */
+#define AID_INTEL_SEC     3701  /* intel security device */
 
 #define AID_MISC          9998  /* access to misc storage */
 #define AID_NOBODY        9999
@@ -164,6 +165,7 @@ static const struct android_id_info android_ids[] = {
     { "net_bw_stats",  AID_NET_BW_STATS, },
     { "net_bw_acct",   AID_NET_BW_ACCT, },
     { "net_bt_stack",  AID_NET_BT_STACK, },
+    { "intel_sec",     AID_INTEL_SEC, },
 
     { "misc",          AID_MISC, },
     { "nobody",        AID_NOBODY, },
-- 
1.7.4.1

